testclient.cpp

```diff
@@ -8,6 +8,9 @@
  *****************************************************************************/
 #include <QCoreApplication>
 #include <QStringList>
+#ifdef __HAIKU__
```

Comment:
`Q_OS_HAIKU`

Reply:
Right, ideally we can fix and drop this anyway.